youtube_dl/extractor/youtube.py

```diff
@@ -889,6 +892,14 @@ def _real_extract(self, url):
             'http' if self._downloader.params.get('prefer_insecure', False)
             else 'https')
 
+        start_time = None
+        parsed_url = compat_urllib_parse_urlparse(url)
+        for component in [parsed_url.fragment, parsed_url.query]:
+            query = compat_parse_qs(component)
+            if 't' in query:
```

Comment:
It also recognizes it as `start`. Moreover it supports `end` counterpart (e.g. http://www.youtube.com/v/4OuHSDsuhXc?start=10&end=25) that would be reasonable to add as well hoping youtube-dl will support downloading fragments some day.